test/frontend-integration-test/helloworld.spec.js

```diff
@@ -24,7 +24,7 @@ const runWithoutExperimentName = 'helloworld-2-' + Date.now();
 const runWithoutExperimentDescription =
   'test run without experiment description ' + runWithoutExperimentName;
 const waitTimeout = 5000;
-const outputParameterValue = 'Hello world in test';
```

Comment:
The failed e2e test makes me check the case again, and I found that the `outputParameterValue` should be changed to "hello world".

The run created by the example yaml file:

![image](https://user-images.githubusercontent.com/56132941/227340187-2de63dc6-7bf8-4b1e-bce9-2f58a4e7c13d.png)

Reply:
That is interesting, I think the run parameter `hello world` hasn't changed since the beginning: https://github.com/kubeflow/pipelines/blob/master/test/frontend-integration-test/helloworld.yaml#L25. Why does it cause issues only recently?